src/uct/ib/mlx5/rc/rc_mlx5_common.h

```diff
@@ -132,6 +132,15 @@ enum {
     UCT_RC_MLX5_CQE_APP_OP_TM_CONSUMED_MSG   = 0xA
 };
 
+enum {
+    /* IBTA-compliant ordering semantics */
+    UCT_IB_MLX5_ORDERING_IBTA,
```

Comment:
i think it's best to put explicit values 0, 1 and 2 as they are specified in the PRM

Reply:
Agree